server/permissions.py

```diff
@@ -0,0 +1,7 @@
+import enum
+
+
+class Permissions(enum.Enum):
```

Comment:
I've made a choice to delete the file whatsoever for the time being since it's not used anywhere yet anyway and won't likely to be used in this state anytime soon at all :P